lib/active_fedora/orders/association.rb

```diff
@@ -11,6 +11,14 @@ def reader(*args)
       super
     end
 
+    # Meant to override all nodes with the given nodes.
+    # @param [Array<ActiveFedora::Base>] nodes Nodes to set as ordered members
+    def target_writer(nodes)
```

Comment:
No. replace is for replacing the proxies, this is for replacing the nodes the proxies point at.